pkg/proc/variables.go

```diff
@@ -1244,6 +1247,41 @@ func (v *Variable) maybeDereference() *Variable {
 	}
 }
 
+// loadPtr assumes that v is a pointer and loads its value. v also gets a child
+// variable, representing the pointed-to value. If v is already loaded,
+// loadPtr() is a no-op.
+func (v *Variable) loadPtr() {
+	if v.loaded {
```

Comment:
I think checking `v.loaded` might be the wrong thing to check here, since it complicates the code in `loadValueInternal` a bit (adding the defer and special casing the `time.Time` case). Perhaps we just check if this variable already has children since no matter what after this method is called v will have `len(v.Children) > 0`.

Reply:
Done.